src/app/public/modules/error/error-modal.service.ts

```diff
@@ -14,6 +14,10 @@ import {
   SkyErrorModalFormComponent
 } from './error-modal-form.component';
 
+/**
+ * Opens a modal to display a SKY UX-themed error message.
```

Comment:
I _think_ this description should be split up between the service and the `open` method. I think we should leave this first sentence in place here but delete the second sentence and add `Specifies the text to display in the error message.` as the description for the `open` method in the table. @Blackbaud-AlexKingman, does that sound right to you? Or am I misunderstanding this?

Reply:
Yeah, that's the right approach. I'd also add that the config parameter updates more than the text - it can change the close text, description, and title. Could the description be more general to cover all that?

Reply:
So how about `Opens a modal to display a SKY UX-themed error message.` here and `Specifies text for the the error message, including title, description, and action label.` for the `open` method. Do you think that addresses your feedback, @Blackbaud-AlexKingman?